analytics/forecasting.md

```diff
@@ -0,0 +1,30 @@
+# Time Series Forecasting
+
+With past data as training data, it creates a model that predicts future value for a specific date and plot the predicted values on a Time Series Chart.
+
+## Input Data
+Input data should be a time series data. Each row should represent one observation (e.g. one row per date/time). It should have following columns.
```

Comment:
"One row per date/time" is not really true. We can handle multiple rows with a same time/date by aggregation.
We can just say, `Each row should represent one observation with date/time.`